templates/restaurants/list.html

```diff
@@ -61,4 +114,20 @@ <h3>No restaurants available at the moment. Please check back later!</h3>
     </span>
 </div>
 
+    <script>
+        document.getElementById('spotlight-toggle').addEventListener('change', function() {
+        const allRestaurantsDiv = document.getElementById('all-restaurants'); 
+        const spotlightDiv = document.getElementById('spotlight-restaurants'); 
+
+        if (this.checked) {
+            
+                allRestaurantsDiv.style.display = 'none';
+                spotlightDiv.style.display = 'block';
+            } else {
+                allRestaurantsDiv.style.display = 'block';
+                spotlightDiv.style.display = 'none';
+            }
+        });
+    </script>
```

Comment:
This is not the correct way to show filtered restaurants.
You should use a package called django_filters to do the filtering.
https://django-filter.readthedocs.io/en/stable/guide/usage.html

When user enables the Spotlight button, the URL should point to RestaurantsList view and the URL will have `?spotlight=true` in the end. That will filter the restaurants list and return results.